app/common/services/maps.js

```diff
@@ -58,7 +58,8 @@ function (
                 '<strong><a href="/posts/' + feature.properties.id + '">' +
                 title +
                 '</a></strong>' +
-                '<p>' + description + '</p>'
+                '<p>' + description + '</p>' +
+                '<p><b>Collection: </b>' + feature.properties.set_name + '</p>'
```

Comment:
Could you use `collections.join(', ')` so they're joined with a space after the comma too?